models/repo/repo_unit.go

```diff
@@ -236,7 +236,7 @@ func (cfg *ProjectsConfig) GetProjectsMode() ProjectsMode {
 		return cfg.ProjectsMode
 	}
 
-	return ProjectsModeNone
```

Comment:
Okay, this change is weird.
The previous one was really faulty?
Because from the looks of it, it looks more correct than the "correct" one…

Reply:
ProjectsMode is empty when newly creating a ProjectsConfig on a repo with enabled projects, like when the repo was created before this change - there, we need to allow all projects. When the repo unit is disabled, the projects config is newly created in the models/repo/repo.go file, where it is now being manually set to none.